pkg/languages/defaults/python_test.go

```diff
@@ -65,7 +65,38 @@ func TestPythonExtractor_ReadDefaults(t *testing.T) {
 				"ENTRYPOINT": "foo.py",
 			},
 			wantErr: false,
-		}, {
+		},
+		{
+			name: "extract python file containing the string \"__name__=='__main__'\" as the entrypoint",
+			args: args{
+				r: reporeader.TestRepoReader{
+					Files: map[string][]byte{
+						"foo.py": []byte("print('hello world')"),
+						"bar.py": []byte("if __name__ == '__main__': print('hello world')"),
```

Comment:
your regex appears to account for the possibility that the `__main__` string is enclosed in double quotes, so adding a test case for that would ensure functionality too

Reply:
Added corresponding test case for ` __main__` enclosed in double quotes and variable spacing.